src/mmrelay/cli.py

```
@@ -477,7 +477,9 @@ def _validate_e2ee_config(
         if store_path:
             expanded_path = os.path.expanduser(store_path)
             if not os.path.exists(expanded_path):
-                print(f"ℹ️  Note: E2EE store directory will be created: {expanded_path}")
+                print(
+                    f"ℹ️  Note: E2EE store directory will be created: {expanded_path}"
```

Ruff (0.14.13)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)